src/components/Accordion.js

```diff
@@ -1,75 +1,98 @@
 const React = require("react");
 const { DOM: dom, PropTypes } = React;
-
-const { div, span } = dom;
 const Svg = require("./utils/Svg");
 
 require("./Accordion.css");
 
 const Accordion = React.createClass({
   propTypes: {
-    items: PropTypes.array
+    children: PropTypes.array.object
   },
 
   displayName: "Accordion",
 
-  getInitialState: function() {
-    return { opened: this.props.items.map(item => item.opened),
-      created: [] };
+  render() {
+    return dom.div(
+      { className: "accordion" },
+      this.props.children
+    );
+  }
+});
+
+const AccordionPane = React.createClass({
+  propTypes: {
+    item: PropTypes.object,
+    opened: PropTypes.bool,
+    children: PropTypes.array,
+    buttons: PropTypes.object,
+    header: PropTypes.string
   },
 
-  handleHeaderClick: function(i) {
-    const opened = [...this.state.opened];
-    const created = [...this.state.created];
-    const item = this.props.items[i];
+  displayName: "AccordionPane",
+
+  getInitialState: function() {
```

Comment:
I'm curious why you would store the state in the component when you have Redux, rather than providing an isOpened, and toggleOpen functions as props, and then maintain that state in the store.